.gitignore

```diff
@@ -4,12 +4,13 @@ logs
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+.idea
 
 # Node
 node_modules
 
 # Application
-data
+data*
```

Comment:
Is the star necessary? I'm just ignoring the entire data folder, this is ignoring any files prefixed with data?